pyproject.toml

```diff
@@ -21,7 +21,7 @@ where = ["src"]
 
 [project]
 name = "django_pg_migration_tools"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
I'd encourage the previous commit updating dependencies and the part of this commit that changes the version number be a separate PR dedicated to releasing a new version rather than bundling it with this change.

Reply:
Thanks. Are there any pros/cons to either approach I wouldn't be aware of? I thought that if a release is linked to a single change it would be best to keep it in the same PR so that it's easier to link to the scoped work from the tags  page. Similar to how some repos create a single release PR over multiple branches. Not particularly fussy about that either though.

Reply:
yeah, certainly not a hard requirement, but my own sensibility is that releases should be separate as a general habit.

Reply:
[Here's](https://github.com/kraken-tech/django-pg-migration-tools/pull/19) a separated PR for the release. To be merged after this one is merged.